website/sidebars.json

```diff
@@ -17,6 +17,7 @@
       "sanitization-chain-api",
       "matched-data-api",
       "validation-result-api"
-    ]
+    ],
+    "More": ["faq"]
```

Comment:
Can you also make the migration guide accessible here?
Looks like I completely forgot of it in #948 🤦